R/utils.R

```diff
@@ -336,23 +337,23 @@ prepare_precompiled <- function(cpp_options = list(), quiet = FALSE) {
 NULL
 
 #' @rdname stan_threads
+#' @deprecated
 #' @export
 #' @return The value of the environment variable `STAN_NUM_THREADS`.
 num_threads <- function() {
+  warning("'stan_threads()' is deprecated. Please use the 'sampling_info()' method of the fit object to obtain the number of threads used.")
```

Comment:
It should say `num_threads()` is deprecated not  `stan_threads()`, right?

Reply:
Fixed. Thanks!